### ISSUE TITLE
Application crashes constantly, when network location provider is unavailable

### ISSUE THREAD

**Issue by scento** [SecUpwN/Android-IMSI-Catcher-Detector#326]
I experience constant crashes, as long as the location tracking service of AIMSICD is active. This seems to be related to [this line](https://github.com/SecUpwN/Android-IMSI-Catcher-Detector/blob/master/app/src/main/java/com/SecUpwN/AIMSICD/service/LocationTracker.java#L102) of code.

**Versions:**
* Device: Hammerhead (Nexus 5, GSM)
* RIL: 56
* AIMSICD: 0.1.25-alpha-b15 *(self-compiled)*
* ROM: AOSP 4.4.4 *(self-compiled)*, no GApps
* XPosed 2.6.1
* XPrivacy 3.6.2 *(not restricting AIMSICD)*

**Logcat:**
```
E/AndroidRuntime(25296): Process: com.SecUpwN.AIMSICD, PID: 25296
E/AndroidRuntime(25296): java.lang.IllegalArgumentException: provider doesn't exist: network
E/AndroidRuntime(25296):        at android.os.Parcel.readException(Parcel.java:1469)
E/AndroidRuntime(25296):        at android.os.Parcel.readException(Parcel.java:1419)
E/AndroidRuntime(25296):        at android.location.ILocationManager$Stub$Proxy.requestLocationUpdates(ILocationManager.java:540)
E/AndroidRuntime(25296):        at android.location.LocationManager.requestLocationUpdates(LocationManager.java:860)
E/AndroidRuntime(25296):        at android.location.LocationManager.requestLocationUpdates(LocationManager.java:454)
E/AndroidRuntime(25296):        at com.SecUpwN.AIMSICD.service.LocationTracker.start(SourceFile:55)
E/AndroidRuntime(25296):        at ph.run(SourceFile:125)
E/AndroidRuntime(25296):        at pg.onSensorChanged(SourceFile:78)
E/AndroidRuntime(25296):        at android.hardware.SystemSensorManager$SensorEventQueue.dispatchSensorEvent(SystemSensorManager.java:418)
E/AndroidRuntime(25296):        at android.os.MessageQueue.nativePollOnce(Native Method)
E/AndroidRuntime(25296):        at android.os.MessageQueue.next(MessageQueue.java:138)
E/AndroidRuntime(25296):        at android.os.Looper.loop(Looper.java:123)
E/AndroidRuntime(25296):        at android.app.ActivityThread.main(ActivityThread.java:5001)
E/AndroidRuntime(25296):        at java.lang.reflect.Method.invokeNative(Native Method)
E/AndroidRuntime(25296):        at java.lang.reflect.Method.invoke(Method.java:515)
E/AndroidRuntime(25296):        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:785)
E/AndroidRuntime(25296):        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:601)
E/AndroidRuntime(25296):        at de.robv.android.xposed.XposedBridge.main(XposedBridge.java:132)
E/AndroidRuntime(25296):        at dalvik.system.NativeStart.main(Native Method)
W/ActivityManager(14332):   Force finishing activity com.SecUpwN.AIMSICD/.AIMSICD
```

**Comment by scento** [comment]
I realized that this was already fixed in *0.1.25-alpha-b28* 0ca39a20518c6d0f9fd174b8758c4e5661b3a6e8.

**Comment by E3V3A** [comment]
Thanks @scento for keeping an eye and reporting! I'm glad you found it was fixed even though we haven't made any release lately. Keep up the testing.
